frame/vamm/src/tests/mod.rs

```diff
@@ -110,6 +118,14 @@ fn create_vamm(vamm_config: &VammConfig<Balance, Moment>) {
 	assert_ok!(TestPallet::create(vamm_config));
 }
 
+fn twap_update_delay(vamm_id: VammId) -> Moment {
+	let vamm_state = TestPallet::get_vamm(vamm_id).unwrap();
+	vamm_state
+		.twap_period
+		.saturating_add(vamm_state.twap_timestamp)
+		.saturating_add(1)
+}
```

Comment:
Curious to know the reason behind the + 1 here

Reply:
For the caller to be able to update the twap successfully, it has to call it strictly after `twap_period`. As I'm using this function as a helper to jump to the next available time to update twaps in tests, I simply added this `+ 1` so I don't need to do so in the tests itself.